packages/tools/devtools/devtools-core/src/messaging/Utilities.ts

```diff
@@ -106,18 +106,27 @@ export function handleIncomingMessage(
 ): void {
 	// TODO: remove loggingOptions once things settle.
 
-	if (message === undefined || !isDevtoolsMessage(message)) {
+	const loggingPreamble =
```

Comment:
While I was here, I figured I would improve the logging here a bit so we can tell when messages are being dropped and _why_.